rebuild-sites-amplify/index.js

```diff
@@ -0,0 +1,52 @@
+const {
+  AmplifyClient,
+  StartJobCommand,
+  ListAppsCommand,
+  UpdateAppCommand,
+} = require("@aws-sdk/client-amplify");
+const path = require("path");
+const fs = require("fs");
+const awsClient = new AmplifyClient({
+  region: "ap-southeast-1",
+  accessKeyId: process.env.AWS_ACCESS_KEY_ID,
+  secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
+});
+
+const MAX_RESULTS_PER_PAGE = 100
+
+async function rebuildAllAmplifySites() {
+  let sitesList = {}
+  let nextToken
+  const listCommand = new ListAppsCommand({
+    maxResults: MAX_RESULTS_PER_PAGE,
+  })
+  response = await (awsClient.send(listCommand))
```

Comment:
are we not missing a declaration here

Reply:
https://github.com/isomerpages/isomer-tooling/pull/43/commits/25e251eee1d15d34679752e2341314172fa9b0c1